### PR TITLE
Add explicit 'localhost' to tests involving server

### PR DESCRIPTION
**Pull request by nichite**
This was breaking internal tests when not specified. Also, remove an unnecessary import, and run dartfmt on the whole repo.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## 2.1.3
+
+* Fix: explicitly specify `'localhost'` for round trip tests
+* Remove unnecessary import of HTTP2 lib
+* Run `dartfmt`
```
Comment (sigurdm) [inline CHANGELOG.md]: Keep the Changelog to user-relevant changes (IE. only the first item)

Comment (nichite) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+## 2.1.3
+
+* Fix: explicitly specify `'localhost'` for round trip tests
```
Comment (sigurdm) [inline CHANGELOG.md]: Reading this again, this is just a test.
I think we can skip publishing this in a version for itself.

Comment (nichite) [reply CHANGELOG.md]: Sure, reverted.

**Pull request merged**